homeassistant/components/snooz/translations/en.json

```diff
@@ -0,0 +1,28 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured",
+            "already_in_progress": "Configuration flow is already in progress",
+            "no_devices_found": "No devices found on the network"
+        },
+        "error": {},
```

Comment:
Looks like there are some stale values

You can regenerate this with `python3 -m script.translations develop`